kotlinx-coroutines-core/api/kotlinx-coroutines-core.api

```diff
@@ -62,6 +62,7 @@ public class kotlinx/coroutines/CancellableContinuationImpl : kotlin/coroutines/
 	public fun getContinuationCancellationCause (Lkotlinx/coroutines/Job;)Ljava/lang/Throwable;
 	public final fun getResult ()Ljava/lang/Object;
 	public fun getStackTraceElement ()Ljava/lang/StackTraceElement;
+	public final fun getState$kotlinx_coroutines_core ()Ljava/lang/Object;
```

Comment:
Please investigate why these getters are now considered public. It seems like a specific attribute is missing from metadata